app/src/main/res/layout/item_conversation_detail_left.xml

```diff
@@ -23,18 +23,18 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginEnd="@dimen/dp_56"
-        android:layout_marginLeft="@dimen/dp_8"
         android:layout_marginRight="@dimen/dp_56"
-        android:layout_marginStart="@dimen/dp_8"
-        android:layout_marginTop="@dimen/dp_16">
+        android:layout_marginTop="@dimen/dp_8">
 
         <ImageView
             android:id="@+id/image_recording_audio"
             android:layout_width="@dimen/dp_24"
             android:layout_height="@dimen/dp_24"
-            android:background="@drawable/ic_audio_conversation"
+            android:background="@drawable/bg_menu_media_creating_conversation"
             android:contentDescription="@string/description_recording_audio_convention"
             android:onClick="@{() -> viewModel.onMediaItemClick(conversation)}"
+            android:padding="@dimen/dp_4"
+            android:src="@drawable/ic_menu_media"
```

Comment:
ERROR: [VectorDrawableCompat] When using VectorDrawableCompat, you need to use `app:srcCompat`.
 To use VectorDrawableCompat, you need to make two modifications to your project. First, set `android.defaultConfig.vectorDrawables.useSupportLibrary = true` in your `build.gradle` file, and second, use `app:srcCompat` instead of `android:src` to refer to vector drawables.